### DISCUSSION TITLE
Sent crypto on the wrong network seeking refund

### DISCUSSION THREAD

**Discussion by MR-SkootzZz** [blockscout/blockscout#11329] [Support]
I am a newbie and didn't DBL check my networks before transferring usdt to wallet. How can I contact the receiver to see in good faith they will do a refund

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
